mm2src/coins/nft/storage/sql_storage.rs

```diff
@@ -22,7 +22,7 @@ use std::sync::{Arc, Mutex};
 
 fn nft_list_table_name(chain: &Chain) -> String { chain.to_ticker() + "_nft_list" }
 
-fn nft_tx_history_table_name(chain: &Chain) -> String { chain.to_ticker() + "_nft_tx_history" }
+fn nft_transfer_history_table_name(chain: &Chain) -> String { chain.to_ticker() + "_nft_transfer_history" }
```

Comment:
Any chain ticker starting with a number will cause the sql statement that creates the table to fail unexpectedly.

Reply:
I realize this isn't directly related to the changes in this PR. However, if we are still in the process of restructuring the DB, please consider https://github.com/KomodoPlatform/komodo-defi-framework/issues/1916

Reply:
It's not a problem in this case since we use hardcoded chain tickers for NFTs
https://github.com/KomodoPlatform/komodo-defi-framework/blob/92372cbf73422a4c65e6c65d3a9f7b744f6dbc76/mm2src/coins/nft/nft_structs.rs#L71-L81 but this should be solved in a general way for other coins tables such as 1inch.